ova/generate_ova.sh

```diff
@@ -55,163 +66,203 @@ clean() {
-
                 export OPENDISTRO_VERSION="$2"
                 HAVE_OPENDISTRO_VERSION=true
             else
                 logger "ERROR Need opendistro version."
+                echo "ERROR Need opendistro version."
                 help 1
             fi
             shift 2
         ;;
 
         "-f" | "--filebeat")
             if [ -n "$2" ]; then
-
                 export ELK_VERSION="$2"
                 HAVE_ELK_VERSION=true
             else
                 logger "ERROR: Need filebeat version."
+                echo "ERROR: Need filebeat version."
                 help 1
             fi
             shift 2
         ;;
 
         "-r" | "--repository")
             if [ -n "$2" ]; then
-
-                export PACKAGES_REPOSITORY="$2"
-                HAVE_PACKAGES_REPOSITORY=true
+                PACKAGES_REPOSITORY="$2"
+                shift 2
             else
-                logger "ERROR: package repository is needed."
+                logger "ERROR: Value must be: [prod/dev]"
+                echo "ERROR: Value must be: [prod/dev]"
                 help 1
             fi
-            shift 2
         ;;
 
         "-u" | "--ui")
             if [ -n "$2" ]; then
                 UI_REVISION="$2"
+                shift 2
             else
-                logger "ERROR: package repository is needed."
                 help 1
             fi
-            shift 2
         ;;
-        "-b"|"--branch")
-            if [ -n "$2" ]; then
 
+        "-b" | "--branch")
+            if [ -n "$2" ]; then
                 BRANCH="$2"
                 shift 2
             else
-                logger "ERROR: Need branch to build."
+                logger "ERROR: Need a value"
+                echo "ERROR: Need a value"
+                help 1
+            fi
+        ;;
+
+        "-d" | "--doc")
+            if [ -n "$2" ]; then
+                BRANCHDOC="$2"
+                shift 2
+            else
+                logger "ERROR: Need a value"
+                echo "ERROR: Need a value"
                 help 1
             fi
-            ;;
-        "-s"|"--store-path")
+        ;;
+
+        "-s" | "--store-path")
             if [ -n "$2" ]; then
                 OUTPUT_DIR="$2"
                 shift 2
             else
                 logger "ERROR: Need store path"
+                echo "ERROR: Need store path"
+                help 1
+            fi
+        ;;
+
+        "-g" | "--debug")
+            if [ -n "$2" ]; then
+                DEBUG="$2"
+                shift 2
+            else
+                logger "ERROR: Need a value [yes/no]"
+                echo "ERROR: Need a value [yes/no]"
                 help 1
             fi
-            ;;
+        ;;
+
         "-c"|"--checksum")
             if [ -n "$2" ]; then
                 CHECKSUM_DIR="$2"
                 CHECKSUM="yes"
                 shift 2
             else
-                CHECKSUM="yes"
+                CHECKSUM="no"
```

Comment:
Fix description: 
- "else"  is necessary to report an error in case the user uses the checksum parameter without value.
- Default checksum path is set to "${scriptpath}/checksum"
- Now the value received by parameter is validated

Fixed in: https://github.com/wazuh/wazuh-packages/pull/763/commits/08a256476d4fd8c3aa575008634a790930fbe3b6